etc/update.sh

```diff
@@ -14,25 +14,27 @@ cat /srv/www/www-mathe-oscar/data/webhook.secret >> .htaccess
 bundle config set --local path 'vendor/bundle'
 bundle install
 
+# use the venv defined in .venv
+# set this up before running this script!
+echo "Activating venv..."
```

Comment:
can't you add a line before this doing roughly `if .venv is not an existing dir, set up a venv` ?

That will make it that much easier to set up a new web server if we ever have to. The alternative would be to add this as another step in `etc/README.md` (I think adding it to the script is preferable to that)